scripts/install_theme.sh

```diff
@@ -2,13 +2,9 @@
 
 set -e
 
-if [ ! -d "mkdocs-material" ] ; then
-  git clone --depth=1 https://github.com/OI-wiki/mkdocs-material.git
+THEME_NAME="mkdocs-material"
+if [ ! -d "$THEME_NAME" ] ; then
+  git clone --depth=1 ${THEME_REPO:-"https://github.com/Menci/mkdocs-material.git"} "$THEME_NAME"
```

Comment:
```suggestion
  git clone --depth=1 ${THEME_REPO:-"https://github.com/OI-wiki/mkdocs-material.git"} "$THEME_NAME"
```

After review, change default theme repo back before merging.